2021-12-18 04:52:43,077 INFO    MainThread:77653 [wandb_setup.py:_flush():69] setting env: {}
2021-12-18 04:52:43,077 INFO    MainThread:77653 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-18 04:52:43,077 INFO    MainThread:77653 [wandb_init.py:_log_setup():336] Logging user logs to /home/jhmoon/mlp-mixer-pytorch/mlp_mixer_pytorch/wandb/run-20211218_045243-5uhwikt3/logs/debug.log
2021-12-18 04:52:43,077 INFO    MainThread:77653 [wandb_init.py:_log_setup():337] Logging internal logs to /home/jhmoon/mlp-mixer-pytorch/mlp_mixer_pytorch/wandb/run-20211218_045243-5uhwikt3/logs/debug-internal.log
2021-12-18 04:52:43,077 INFO    MainThread:77653 [wandb_init.py:init():369] calling init triggers
2021-12-18 04:52:43,078 INFO    MainThread:77653 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'t2i': True, 'i2t': False, 'eval_len_size': 354, 'CXRBERT': False, 'do_train': True, 'do_test': False, 'eval_during_training': True, 'MIMIC_dset': True, 'label_conditioned': True, 'train_dataset': '/home/data_storage/mimic-cxr/dataset/image_preprocessing/Train.jsonl', 'label_conditioned_valid_dataset': '/home/data_storage/mimic-cxr/dataset/dataset/retrieval/dset/T2I_Label_Valid.jsonl', 'label_conditioned_test_dataset': '/home/data_storage/mimic-cxr/dataset/dataset/retrieval/dset/T2I_Label_Test.jsonl', 'studyID_valid_dataset': '/home/data_storage/mimic-cxr/dataset/retrieval/openi/T2I_ID_Test_openi.jsonl', 'studyID_test_dataset': '/home/data_storage/mimic-cxr/dataset/retrieval/openi/T2I_ID_Test_openi.jsonl', 'output_path': 'output/2021-12-18 04:52:42.630480', 'with_cuda': True, 'cuda_devices': None, 'epochs': 50, 'batch_size': 128, 'num_workers': 0, 'hidden_size': 768, 'embedding_size': 768, 'weight_load': True, 'load_pretrained_model': '/home/data_storage/mimic-cxr/models/retrieval_openi/openi_base_sc_par_label', 'bert_model': 'bert-base-scratch', 'vocab_size': 30522, 'img_postion': True, 'seq_len': 128, 'max_seq_len': 512, 'img_hidden_sz': 2048, 'img_encoder': 'full-fiber', 'img_channel': 1, 'num_image_embeds': 256, 'img_size': 224, 'img_embed_pool_type': 'max', 'lr': 1e-05, 'gradient_accumulation_steps': 4, 'warmup': 0.1, 'seed': 123, 'warmup_steps': 0, 'dropout_prob': 0.1, 'beta1': 0.9, 'beta2': 0.999, 'eps': 1e-06, 'weight_decay': 0.01}
2021-12-18 04:52:43,078 INFO    MainThread:77653 [wandb_init.py:init():418] starting backend
2021-12-18 04:52:43,078 INFO    MainThread:77653 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-18 04:52:43,091 INFO    MainThread:77653 [backend.py:ensure_launched():123] starting backend process...
2021-12-18 04:52:43,100 INFO    MainThread:77653 [backend.py:ensure_launched():127] started backend process with pid: 77738
2021-12-18 04:52:43,101 INFO    MainThread:77653 [wandb_init.py:init():423] backend started and connected
2021-12-18 04:52:43,104 INFO    MainThread:77653 [wandb_init.py:init():465] updated telemetry
2021-12-18 04:52:43,104 INFO    MainThread:77653 [wandb_init.py:init():484] communicating current version
2021-12-18 04:52:43,738 INFO    MainThread:77653 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-18 04:52:43,739 INFO    MainThread:77653 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-12-18 04:53:13,740 INFO    MainThread:77653 [interface.py:_communicate_run():642] couldn't get run from backend
2021-12-18 04:53:13,740 ERROR   MainThread:77653 [wandb_init.py:init():501] backend process timed out
2021-12-18 04:53:13,740 ERROR   MainThread:77653 [wandb_init.py:init():510] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2021-12-18 04:53:18,782 INFO    MainThread:77653 [wandb_init.py:teardown():187] tearing down wandb.init
